Sub-menu-via-event-target/test.js

```diff
@@ -0,0 +1,20 @@
+const ul = document.querySelector(".tree");
+
+const li = document.querySelectorAll("li");
+
+li.forEach((item) => {
+  const span = document.createElement("SPAN");
+  span.classList.add(".p");
+  item.prepend(span); // добавили перед тегом span
+  span.append(span.nextSibling); // добавили после тега span
+});
+
+ul.addEventListener("click", function (event) {
```

Comment:
Please use arrow func
```suggestion
ul.addEventListener("click", (event) => {
```